### PR TITLE
issue#5363 Ankit8848/update label quit

### PR DESCRIPTION
**Pull request by Ankit8848**
## Checks

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [x] I accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified.

### CODE REVIEW

File: implementations/rust/ockam/ockam_app/src/quit/menu_items.rs [hunk 1]
```diff
@@ -10,7 +10,7 @@ pub struct QuitActions {
 
 impl QuitActions {
     pub fn new() -> QuitActions {
-        let quit = CustomMenuItem::new("quit".to_string(), "Quit").accelerator("cmd+q");
+        let quit = CustomMenuItem::new("quit".to_string(), "Quit Ockam").accelerator("cmd+q");//update label to "Quit Ockam"
```
Comment (mrinalwadhwa) [inline implementations/rust/ockam/ockam_app/src/quit/menu_items.rs]: > //update label to "Quit Ockam"

Could you remove this comment please. The code doesn't need to talk about an old state.

File: implementations/rust/ockam/ockam_app/src/enroll/menu_items.rs [hunk 2]
```diff
@@ -18,7 +18,7 @@ pub struct EnrollActions {
 impl EnrollActions {
     pub fn new() -> EnrollActions {
         let enroll = CustomMenuItem::new(ENROLL_MENU_ID, "Enroll...").accelerator("cmd+e");
-        let reset = CustomMenuItem::new(RESET_MENU_ID, "Reset...").accelerator("cmd+r");
+        let reset = CustomMenuItem::new(RESET_MENU_ID, "Reset").accelerator("cmd+r");// Updated label to "Reset"
```
Comment (mrinalwadhwa) [inline implementations/rust/ockam/ockam_app/src/enroll/menu_items.rs:22]: This change os already merged, please remove it from this commit.
You may need to rebase to our latest develop branch.

File: implementations/rust/ockam/ockam_app/src/quit/menu_items.rs (hunk 1 above)
Comment (Ankit8848) [reply implementations/rust/ockam/ockam_app/src/quit/menu_items.rs]: yeah, sure. @mrinalwadhwa

**Comment by mrinalwadhwa** [comment]
@Ankit8848 thank you for spending time on this.

A few things to improve before we can merge

Note that the PR has three commits in it and one of them is merge commit 22ab7bed07f5da28cafa0deca3ce39bf3839a920

Screenshot 2023-07-16 at 12 19 12 PM

We don't allow merge commits in our repo. To update to the develop branch we use git rebase instead of merge commits. Please read about how that works.

If you can't find a good reference for that please let me know and I'll try to look for a good reference.

If you follow the git rebase workflow the end result should be a single commit in the PR that only changes the label and nothing else.

**Comment by Ankit8848** [comment]
sir, I have to delete this branch locally and again make changes then rebase it and push it . I have to close this pr ? @mrinalwadhwa

**Comment by mrinalwadhwa** [comment]
You can close the PR but it's not necessary. You can also force push to your branch.

**Pull request closed without merging**